docs/lib/auth/fragments/flutter/signout/00_signout_callout.md

```diff
@@ -0,0 +1,3 @@
+<amplify-callout usage>
+Please note that the Android platform will allow a new user to sign in even if a different user has already signed in, while the iOS platform will force any currently authenticated user to signout before a different user can sign in.
```

Comment:
Suggestion to change it to an action required on the developers part: "On iOS, if a user is signed in, you would have to explicitly call signOut before a new user can sign in."

Reply:
Also think it would be good to clarify that Android doesn't allow multiple users to be signed in at the same time but rather it replaces the previous sign in. Maybe something like this:

"Please note that to sign in a new user, you must first sign out of your existing account on iOS. On Android, this is not required and you can simply sign in the new user without first signing out."

Reply:
@Ashish-Nanda I think @TrekSoft 's suggestion will satisfy yours as well.